website/blog/2020-06-19-getting-started-and-aria.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Getting Started with PrestoDB and Aria Scan Optimizations
+author: Adam Shook
+authorURL: https://www.linkedin.com/in/adamjshook
+---
+
+This article was originally published by Adam on June 15th, 2020 over at his blog at [datacatessen.com](https://datacatessen.com/blog/prestodb-aria/).
+
+---
+
+[PrestoDB](https://prestodb.io) recently released a set of experimental features under their Aria project in order to increase table scan performance of data stored in ORC files via the Hive Connector.  In this post, we'll check out these new features at a very basic level using a test environment of PrestoDB on Docker.  To find out more about the Aria features, you can check out the [Facebook Engineering](https://engineering.fb.com/data-infrastructure/aria-presto/) blog post which was published June 2019.
+
```

Comment:
Can you add the truncate tag here so it doesn't display the whole post in the blog index?